src/node_http_parser_impl.h

```diff
@@ -825,11 +825,20 @@ class Parser : public AsyncWrap, public StreamListener {
     got_exception_ = false;
   }
 
-
-  int TrackHeader(size_t len) {
+  enum tracking_position {
+    before_headers,
+    after_request_line
+  };
```

Comment:
Style: idiomatic Node.js style is `TrackingPosition` and `kBeforeHeaders` or `BEFORE_HEADERS`; the former is arguably the most common.

There should be a blank line before the method.

Apropos the enum name, maybe `HeaderParseState` (or something like it) is a better name?